hooks/useVotingPlugins.ts

```diff
@@ -137,6 +141,9 @@ export function useVotingPlugins() {
         if (HELIUM_VSR_PLUGINS_PKS.includes(currentPluginPk.toBase58())) {
           handleSetHeliumVsrClient(wallet, connection, currentPluginPk)
         }
+        if (PYTH_PLUGIN_PK.includes(currentPluginPk.toBase58())) {
```

Comment:
there is really no reason that all the plugins should be in the same `useEffect`, and many reasons they should not be. Please just make one `useEffect` for initializing your plugin. And please give that `useEffect` exhaustive dependencies.

Reply:
addressed